src/test/resources/org/broadinstitute/hellbender/engine/example_variants.vcf

```diff
@@ -0,0 +1,14 @@
+##fileformat=VCFv4.1
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
```

Comment:
Erm, this is just a slightly modified version of `feature_data_source_test.vcf`, which you are using as the auxiliary input to your test. At least change the locations and alleles....